### ISSUE TITLE
Generate social preview automatically

### ISSUE THREAD

**Issue by trustin** [line/armeria#3469]
We currently use a manually rendered static social preview image for our repository. We could make use of some 'social preview generator' to generate a nicer one, e.g.

![image](https://user-images.githubusercontent.com/173918/115531512-d3c89500-a2cf-11eb-8fc6-106d274dc913.png)

![image](https://user-images.githubusercontent.com/173918/115531543-dd51fd00-a2cf-11eb-8fc3-fa88d6f09791.png)

**Comment by syleeeee** [comment]
So will it be automatically generated when we remove the social preview image? 
I tried with Central Dogma(without social preview settings) and it looks like this. 
![image](https://user-images.githubusercontent.com/6274210/117293528-737b4b00-aeac-11eb-86b9-8061cf192c33.png)

**Comment by trustin** [comment]
Yeah, it seems so. It'd be nice if we can set the logo at the project level instead of using the LINE logo.

**Comment by syleeeee** [comment]
Well, I searched it but I think we can't set the logo at the project level so far. 

So how about now? Is this better? 
![image](https://user-images.githubusercontent.com/6274210/117649483-894d8080-b1ca-11eb-81d2-a2fb01aa93f2.png)

**Comment by trustin** [comment]
Hard to say :laughing: How about re-designing our social image so it contains some information in addition to the logo? It looks somewhat vague at the moment.

**Comment by syleeeee** [comment]
🤣  Agreed, but the default social image shows some real-time numbers such as the number of contributors or stars. 
Well, I'll let the image empty and think about it.
